Mathlib/CategoryTheory/Limits/Shapes/Biproducts.lean

```diff
@@ -421,6 +421,22 @@ theorem biproduct.ι_π_ne (f : J → C) [HasBiproduct f] {j j' : J} (h : j ≠
     biproduct.ι f j ≫ biproduct.π f j' = 0 := by simp [biproduct.ι_π, h]
 #align category_theory.limits.biproduct.ι_π_ne CategoryTheory.Limits.biproduct.ι_π_ne
 
+-- The `simpNF` linter incorrectly identifies these as simp lemmas that could never apply.
```

Comment:
Is there a GitHub issue or Zulip thread tracking this issue that you could link to here? We should make sure that we don't forget about this.

Reply:
Added a link to the generic simpNF tracking issue.